tests/unit/client/Landing.test.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import * as rdd from 'react-device-detect'
+
+import { LandingPage } from '@/components/LandingPage'
+
+describe('<LandingPage />', () => {
+  it('renders the LandingPage component', () => {
+    render(<LandingPage />)
+  })
+  it('<LandingPage />in mobile view', () => {
+    Object.assign(rdd, { isMobile: true })
```

Comment:
This assign is doing nothing except making a new object that is not used. With this approach I assume you are trying to [mock an import](https://dev.to/lmeromy/testing-react-components-how-to-mock-imports-with-jest-1k90) in `<LandingPage />`?